Draft: Write introduction, add implementation (#14)
dominikbell/cemracs_paper: main.tex

\begin{abstract}

> In practice, there are various ways to solve gyro-kinetic equations. Here, we will combine the Semi-Lagrangian (SL) method and the Arakawa (AKW) scheme combined with a time-integrator. Both methods are successfully used in practice when applied individually, in our case, we combine them by first decomposing the problem into a fast (parallel) and a slow (perpendicular) dynamical system. The SL approach will be used to solve the fast and the AKW scheme for the slow subsystem. So far in our reference code \cite{pygyro_code}, the entire model is solved only using the SL method. Our goal is to replace the discretization of the slow subsystem by the AKW scheme, in order to improve preservation of conserved quantities of the physical system at hand. \\

Reviewer: ```suggestion
In practice, there are various ways to solve gyro-kinetic equations. Here, we will combine the Semi-Lagrangian (SL) method and the Arakawa (AKW) scheme combined with a time-integrator. Both methods are successfully used in practice when applied individually, in our case, we combine them by first decomposing the problem into a fast (parallel) and a slow (perpendicular) dynamical system. The SL approach will be used to solve the fast subsystem, and the AKW scheme for the slow subsystem. So far in our reference code \cite{pygyro_code}, the entire model is solved only using the SL method. Our goal is to replace the method used in the slow subsystem by the AKW scheme, in order to improve the conservation of the physical constants. \\

```

Not sure about the repetition between the last 2 sentences

Reviewer: Or:
```suggestion
The SL approach and the AKW scheme will be used to solve respectively the fast and the slow subsystems. 
```

> In practice, there are various ways to solve gyro-kinetic equations. Here, we will combine the Semi-Lagrangian (SL) method and the Arakawa (AKW) scheme combined with a time-integrator. Both methods are successfully used in practice when applied individually, in our case, we combine them by first decomposing the problem into a fast (parallel) and a slow (perpendicular) dynamical system. The SL approach will be used to solve the fast and the AKW scheme for the slow subsystem. So far in our reference code \cite{pygyro_code}, the entire model is solved only using the SL method. Our goal is to replace the discretization of the slow subsystem by the AKW scheme, in order to improve preservation of conserved quantities of the physical system at hand. \\

Reviewer: > Here, we will combine the Semi-Lagrangian (SL) method and the Arakawa (AKW) scheme combined with a time-integrator.

Repetition of "combine". Do you mean:
> Here, we will combine the Semi-Lagrangian (SL) method and the Arakawa (AKW) scheme with a time-integrator.

or:

> Here, we will combine the Semi-Lagrangian (SL) method with a subcycling Arakawa (AKW) scheme.

> In practice, there are various ways to solve gyro-kinetic equations. Here, we will combine the Semi-Lagrangian (SL) method and the Arakawa (AKW) scheme combined with a time-integrator. Both methods are successfully used in practice when applied individually, in our case, we combine them by first decomposing the problem into a fast (parallel) and a slow (perpendicular) dynamical system. The SL approach will be used to solve the fast and the AKW scheme for the slow subsystem. So far in our reference code \cite{pygyro_code}, the entire model is solved only using the SL method. Our goal is to replace the discretization of the slow subsystem by the AKW scheme, in order to improve preservation of conserved quantities of the physical system at hand. \\

Reviewer: > Both methods are successfully used in practice when applied individually,

Not sure what is meant by this

> After an introduction of the gyro-kinetic equations in Section \ref{sec:gk-model}, we will describe this splitting ansatz in combination with the two schemes in Section \ref{sec:splitting_discretization}. Lastly in Section \ref{sec:num_exp}, numerical experiments will be presented, first as a verification of the AKW scheme, and after that the application to the full PyGyro model. Concluding with Section \ref{sec:conclusion}, we discuss the benefits of this approach. 

Reviewer: This line feels like it should be at the end of the introduction not the abstract.

\end{abstract}

\begin{resume}%TODO: verify french
	En pratique, il existe plusieurs façons de résoudre les équations gyro-cinétiques. Ici, nous combinerons la méthode semi-lagrangienne (SL) et le schéma d'Arakawa (AKW) combiné à un intégrateur temporel. Les deux méthodes sont utilisées avec succès dans la pratique lorsqu'elles sont appliquées individuellement. Dans notre cas, nous les combinons en décomposant d'abord le problème en un système dynamique rapide (parallèle) et un système dynamique lent (perpendiculaire). L'approche SL sera utilisée pour résoudre le système rapide et le schéma AKW pour le sous-système lent. Jusqu'à présent, dans notre code de référence \cite{pygyro_code}, le modèle entier est résolu en utilisant uniquement la méthode SL. Notre objectif est de remplacer la discrétisation du sous-système lent par le schéma AKW, afin d'améliorer la préservation des quantités conservées du système physique en question. \\

> 	Après une introduction des équations gyro-cinétiques dans la Section \ref{sec:gk-model}, nous décrirons cet ansatz de fractionnement en combinaison avec les deux schémas dans la Section \ref{sec:splitting_discretization}. Enfin, dans la section \ref{sec:num_exp}, des expériences numériques seront présentées, d'abord pour vérifier le schéma AKW, puis pour l'appliquer au modèle PyGyro complet. Dans la section \ref{sec:conclusion}, nous discutons des avantages de cette approche. 

Reviewer: ```suggestion
	Après une introduction des équations gyro-cinétiques dans la Section \ref{sec:gk-model}, nous décrirons cet ansatz de fractionnement en combinaison avec les deux schémas dans la Section \ref{sec:splitting_discretization}. Enfin, dans la Section \ref{sec:num_exp}, des expériences numériques seront présentées, d'abord pour vérifier le schéma AKW, puis pour l'appliquer au modèle PyGyro complet. Dans la Section \ref{sec:conclusion}, nous discutons des avantages de cette approche. 

```
